### PR TITLE
Use file system to store files

### PR DESCRIPTION
**Pull request by lxylxy123456**
Note that this change requires change in database. 

@rkevin-arch please comment on the way I am handling repeated file name, starting at: https://github.com/lxylxy123456/ngshare/blob/b2a3b1420cf27b42da5e98c7e5350cee6e967d7d/ngshare/ngshare.py#L131

### CODE REVIEW

File: ngshare/ngshare.py [hunk 1]
```diff
@@ -86,7 +96,10 @@ def json_files_pack(self, file_list, list_only):
                 'checksum': i.checksum,
             }
             if not list_only:
-                entry['content'] = base64.encodebytes(i.contents).decode()
+                storage_path = self.application.storage_path
+                actual_path = os.path.join(storage_path, i.actual_name)
+                content = open(actual_path, 'rb').read()
```
Comment (rkevin-arch) [inline ngshare/ngshare.py:101]: might want to close the file afterwards, or just use `with open(...) as f:`

**Comment by rkevin-arch** [comment]
LGTM

**Comment by rkevin-arch** [comment]
The only complaint I have with the way you're dealing with repeated filenames is the `sleep(0.1)`, since it will hang everything for 0.1s. Not sure if you want to use `await` and make it a coroutine or just remove the sleep altogether (uuid4 should be random independent of time). That said, it's only a minor complaint since it should rarely happen.

File: ngshare/ngshare.py (hunk 1 above)
Comment (lxylxy123456) [reply ngshare/ngshare.py:101]: I think `open(...)` is not saved in Python so is auto-closed.

**Comment by lxylxy123456** [comment]
@rkevin-arch I think maybe removing the sleep is better. I am not sure how to use `await` in this situation. I will change the code.

**Pull request merged**